gstreamer-rtsp/src/auto/r_t_s_p_auth_credential.rs

```diff
@@ -0,0 +1,9 @@
+// This file was generated by gir (0113735) from gir-files (???)
+// DO NOT EDIT
+
+use ffi;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::mem;
+use std::ptr;
```

Comment:
Empty file? How did that happen? :)

Reply:
RTSPAuthCredential doesn't have any methods :)

Reply:
Actually I had checked in a bunch of empty files generated by my initial runs of gir, all removed now

Reply:
Even if it had no functions, the struct at least should be here :)